redash/query_runner/presto.py

```diff
@@ -74,11 +88,17 @@ def type(cls):
 
     def get_schema(self, get_stats=False):
         schema = {}
+
+        default_table_schemas = ['pg_catalog', 'information_schema']
+        blacklisted_table_schemas = self.configuration.get('blacklisted_table_schemas', '')
+        final_table_schemas = default_table_schemas + map(str.strip, blacklisted_table_schemas.split(','))
+        table_schemas = ', '.join("'{0}'".format(w) for w in final_table_schemas)
```

Comment:
better names would be
 - `final_table_schemas` -> `exempted_schema_list`
 - `table_schemas` -> `exempted_schema_str`